sos/plugins/openvswitch.py

```diff
@@ -43,6 +43,16 @@ def setup(self):
             "/var/run/openvswitch/ovs-vswitchd.pid"
         ])
 
+        # Gather fdb statistical output for each OVS bridge on the host.
+        br_list_result = self.call_ext_prog("ovs-vsctl list-br")
```

Comment:
`list_br_result` may be a better name here since it matches the syntax of the `ovs-vsctl` command, and would make these lines more readable.

Reply:
Changed